eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -8,13 +8,17 @@ parameters:
       - Name: $(Build.Repository.Name)
         Commitish: $(Build.SourceVersion)
         WorkingDirectory: $(System.DefaultWorkingDirectory)
+  # NOTE: SkipDefaultCheckout is being deprecated in favor of SkipCheckoutNone
```

Comment:
I'm keeping both to make it easier to do PRs one by one in the other repositories. After I'm done, I'll remove this parameter.